doc/src/fix_eco_force.rst

```diff
@@ -0,0 +1,167 @@
+experience a synthetic force as its surrounding is that of an oriented single crystal 
+and thermal fluctuations are masked by the parameter eta. Near the grain boundary 
+however, the gradient is nonzero and synthetic force terms are computed. 
+The orientation file specifies the perfect oriented crystal basis vectors for the 
+two adjoining crystals. The first three lines for the energetically penalized and the 
+last three lines for the energetically favored grain assuming u0 is positive. For 
+negative u0 this is reversed. With the rcut parameter, the size of the region around 
+each atom which is used in the order parameter computation is defined. It should at 
+least include the nearest neighbor shell. For high temperatures or low angle 
+grain boundaries, it might be beneficial to increase rcut in order to get a more 
+precise identification of the atoms surrounding. However, computation time will 
+increase as more atoms considered in the order parameter and force computation. 
+It is also worth noting that the cutoff radius must not exceed the communication 
+distance for ghost atoms in LAMMPS. Currently however, the method stores results 
+for order parameter and force computations in statically allocated arrays to 
+increase efficiency such that the user is limited to 24 nearest neighbors. 
+This is more than enough in most applications. With file, the input file for 
+the 6 oriented crystal basis vectors is specified. Each line of the input file 
+contains the three components of a primitive lattice vector oriented according to 
+the grain orientation in the simulation box. The first (last) three lines correspond 
+to the primitive lattice vectors of the first (second) grain. An example for 
+a :math:`\Sigma\langle001\rangle` misorientation is given at the end.
+
+If no synthetic energy difference between the grains is created, u0=0, the 
+force computation is omitted. In this case, the order parameter of the 
+driving force can be used to track the grain boundary motion throughout the 
+simulation.
+
+
+
+**Restart, fix_modify, output, run start/stop, minimize info:**
+
+No information about this fix is written to :doc: `binary restart files`.
+ 
+The :doc:`fix_modify <fix_modify>` *energy* option is supported by this fix to 
+add the potential energy of atom interactions with the grain boundary 
+driving force to the system's potential energy as part of thermodynamic output. 
+The total sum of added synthetic potential energy is computed and can be accessed 
+by various output options. The order parameter as well as the thermally masked 
+output parameter are stored in per-atom arrays and can also be accessed by various 
+output commands. 
+
+No parameter of this fix can be used with the start/stop keywords of the run command. This fix is
+not invoked during energy minimization.
+
+
+
+Restrictions
+""""""""""""
+
+This fix is part of the MISC package. It is only enabled if LAMMPS was
```

Comment:
as a user contributed style, this should be added to the USER-MISC package instead of MISC. consequently, also the example files should be moved to `examples/USER/misc/<style name>`